include/fmt/chrono.h

```diff
@@ -835,6 +835,9 @@ FMT_CONSTEXPR const Char* parse_chrono_format(const Char* begin,
     case 'T':
       handler.on_iso_time();
       break;
+    case 'P':
+      handler.on_am_pm_lower();
+      break;
```

Comment:
nit: I'd suggest moving this after the 'p' case to keep the standard version first (and similarly for `am_pm_lower` functions below).